gitops/pac/setup.sh

```diff
@@ -73,10 +73,18 @@ mk_tmpdir () {
 }
 
 kustomize () {
-
     GIT_TOKEN=$(echo "${GIT_TOKEN}" | base64)
     WEBHOOK_SECRET=$(echo "${WEBHOOK_SECRET}" | base64)
 
+    # This resource may take some time to spawn, which will cause Kustomize to fail when
```

Comment:
As far as I know, we are not calling this script (gitops/pac/setup.sh) in any other script. So, if we are running this manually, it is expected that you have PaC installed prior to it. And if that's the case, I don't see why we would encounter issues like - "pipelinesascode.tekton.dev" not found in "kubectl api-resources"